README.md

```diff
@@ -53,6 +53,10 @@ const exporter = new DynatraceMetricExporter({
     url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
     APIToken: 'token123'’,
```

Comment:
This might look like something provided automatically. I usually use something like "MY_API_TOKEN" to make clear this is an arbitrary placeholder and not a well-defined env var set by the OneAgent, for example.